packages/terra-action-footer/src/_ActionFooterContainer.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './ActionFooterContainer.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  children: PropTypes.node,
```

Comment:
Can we mark this as `isRequired`? Doesn't make much sense to have a named `Container` without children, right??

Reply:
This is really a wrapper for common styling/behavior between both action footer variations. If there's no actions provided, I still want those styles to apply (empty height of 10px with a top border). No actions === no children

Reply:
👍